core/blockly.js

```diff
@@ -92,25 +93,12 @@ Blockly.selected = null;
 Blockly.draggingConnections = [];
 
 /**
- * Contents of the local clipboard.
- * @type {Element}
- * @private
- */
-Blockly.clipboardXml_ = null;
-
-/**
- * Source of the local clipboard.
- * @type {Blockly.WorkspaceSvg}
- * @private
+ * Get the current contents of the clipboard and associated metadata.
+ * @return {{xml: ?Element, source: ?Blockly.WorkspaceSvg, typeCounts: ?Object}}
```

Comment:
If we could provide a better type for `typeCounts` now that this is public I think that would be really helpful.

Reply:
I switched all of this to `ICopyable.CopyData`, which is an existing type.